pocket_coffea/lib/leptons.py

```diff
@@ -33,6 +33,35 @@ def lepton_selection(events, lepton_flavour, params):
 
     return leptons[good_leptons]
 
+def soft_lepton_selection(events, lepton_flavour, params):
```

Comment:
Another comment on this. 

Looking at the code it seems that this function differs from `lepton_selection` only because of a dxy cut. 
Would it be possible to add a `if  "dxy" in cuts` check also for that and keep a single function?

Reply:
This function has an inverted iso criterium as well, which is a required option, apart from dxy. Would you want to integrate it in the main function?

Reply:
Right! I read too quick. In this case it is fine to keep it separate, it has a different scope